index.html

```diff
@@ -328,11 +338,10 @@ <h1>Uber Navigation</h1>
                         <div class="title">
                             <h1>About Myself</h1>
                         </div>
-                        <div class="text">
+                        <div class="text  bout">
                             <p>Hello I’m a software developer! I can help you build a product , feature or website Look
                                 through some of my work and experience! If you like what you see and have a project you
-                                need
-                                coded, don’t hestiate to contact me.
+                                need coded, don’t hestiate to contact me.
                             </p>
                         </div>
                         <p class="con">LET'S CONNECT</p>
```

Comment:
- [ ] Kindly try to follow the design as much as possible, for the `about me` section, the `font-size` of the text (16px instead of 14px) and the `padding` (of the languages) are different from the ones on Figma. Also, the arrows of the skills section are not aligned under each other, they're kind of cascading.

## Your design:
![image](https://user-images.githubusercontent.com/50721479/194648634-5c561b84-610a-4418-af24-a92ffdcfd76b.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/194648652-38120c39-c6df-40c4-b936-e6faa1c8ede2.png)